src/Geosuggest.jsx

```diff
@@ -399,6 +399,8 @@ class Geosuggest extends React.Component {
       suggestionsList = <SuggestList isHidden={this.state.isSuggestsHidden}
         style={this.props.style.suggests}
         suggestItemStyle={this.props.style.suggestItem}
+        suggestsClassName={this.props.suggestsClassName}
+        suggestItemsClassName={this.props.suggestItemsClassName}
```

Comment:
👍 done.
In the last commit I also updated the README.md with the new property options.